src/pages/Admin/Admin.tsx

```diff
@@ -297,14 +298,14 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
                       interactive={false}
                       resizable={false}
                       iconsMap={
-                        venue.placement && venue.mapIconImageUrl
+                        venue.placement
                           ? {
                               icon: {
                                 width: PLAYA_VENUE_SIZE,
                                 height: PLAYA_VENUE_SIZE,
                                 top: venue.placement.y,
                                 left: venue.placement.x,
-                                url: venue.mapIconImageUrl,
+                                url: DEFAULT_MAP_ICON_URL,
                               },
                             }
                           : {}
```

Comment:
Is `DEFAULT_MAP_ICON_URL` also related to this same concept? Based on the usage here, it seems so. If so, we should probably be removing it from the codebase in this PR as well.

Which means we'll have to change the code here differently.